commands.go

```diff
@@ -2472,6 +2473,18 @@ func (c cmdable) ZInterCard(ctx context.Context, limit int64, keys ...string) *I
 	_ = c(ctx, cmd)
 	return cmd
 }
+func (c cmdable) ZMPop(ctx context.Context, order string, count int64, keys ...string) *ZArrayWithKeyCmd {
```

Comment:
I know we haven't done this previously.. but WDYT about starting to add help strings - and a link to the official command docs at https://redis.io/commands/zmpop. IMHO it helps people understand redis commands.

Reply:
Sure will do that

Reply:
@chayim Can you quote an example for what we should put in the help strings?

Reply:
Yes, we really should add a comment to help users with this command.

Reply:
Added help strings for this commands usage.  Can someone please review the same @monkey92t @chayim